contracts/CMakeLists.txt

```diff
@@ -8,10 +8,10 @@ option(SYSTEM_CONFIGURABLE_WASM_LIMITS
 option(SYSTEM_BLOCKCHAIN_PARAMETERS
        "Enables use of the host functions activated by the BLOCKCHAIN_PARAMETERS protocol feature" ON)
 
-find_package(cdt REQUIRED)
+find_package(cdt)
```

Comment:
any reason to remove `REQUIRED` here (and below for `leap` too)? They are required so seems like they should be marked as such.

Reply:
Got it, will put required back in.